src/components/VideoPlayer/VideoRecommendation.jsx

```diff
@@ -25,8 +29,39 @@ const VideoRecommendation = (msSingleVideoCardDetail) => {
                                         <p className="ms_recommended-video-title ms_fw-b">{eachRecommendation.music_title}</p>
                                         <p className="ms_recommended-video-duration">{eachRecommendation.music_description.music_duration}</p>
                                     </div>
-                                    <div className="ms_recommended-video-icon-container b-rad4">
+                                    <div
+                                        className="ms_recommended-video-icon-container b-rad4"
+                                        onClick={(e) => {
+                                            e.stopPropagation();
+                                            e.preventDefault();
```

Comment:
You can keep only the stop Propagation no need of having the prevent default action

Reply:
hmm yes i've not added functionality to that middle icon yet!!

Reply:
yes actually i was trying to the same thing here, but due to some unknown reason when added e.stopPropagation() then it was causing the page to refresh, and therefore added e.preventDefault() to stop it from refreshing the page!!